commands.go

```diff
@@ -558,7 +567,10 @@ Harsh           @harshb__
 **And many more have joined!**`)
 }
 
-func helpCMD(_ string, u *user) {
+func helpCMD(rest string, u *user) {
+	if rest != "" {
+		return
```

Comment:
I like this form with the return because it lets me keep the indentation on the rest of the code. Could you apply this to the others?